easy_phi/auth.py

```diff
@@ -12,8 +12,10 @@
 import base64
 import functools
 import hashlib
+import re
 import uuid
 import subprocess
+import keyring
```

Comment:
keyring is a package to interface with system keyring (safe password storage). It has a configurable backend, and tries to select best backend by default.
Backend example:  https://en.wikipedia.org/wiki/GNOME_Keyring